src/ussd/v1/ussd.js

```diff
@@ -0,0 +1,336 @@
+'use strict';
+require('dotenv').config();
+const randomstring = require('randomstring');
+
+const db = require('../../../database/models/index.js');
+
+const optsUssd = {
+  schema: {
+    description: 'Ussd Endpoint Route for Farmpro App',
+    tags: ['ussd'],
+    querystring: {
+      type: 'object',
+      required: ['sessionId', 'code', 'phone'],
+      properties: {
+        sessionId: { type: 'string' },
+        code: { type: 'string' },
+        phone: { type: 'string', maxLength: 12, minLength: 12 },
+        text: { type: 'string', default: '' },
+      },
+    },
+
+    response: {
+      200: {
+        description: 'Success Response',
+        content: 'text/plain',
+        type: 'object',
+      },
+      400: {
+        description: 'Error Response',
+        content: 'application/json',
+        type: 'object',
+      },
+    },
+  },
+};
+
+module.exports = function (fastify, options, done) {
+  fastify.get('/ussd', optsUssd, async (request, reply) => {
+    let isRegistered = false;
+    let message = '';
+    let registration_level = 0;
+    const sessionId = request.query.sessionId;
+    const serviceCode = request.query.code;
+    const phoneNumber = request.query.phone;
+    const text = request.query.text;
+    console.log('text-', text);
+    let txt = request.query.text.split(/[*#]/);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['txt' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/BuildForSDG/Farm-Pro/pullRequest?prid=5674718)